orangecontrib/network/widgets/OWNxAnalysis.py

```diff
@@ -249,13 +254,12 @@ def set_items(self, items):
         self.mutex.unlock()
 
     def analyze(self):
-        if self.graph is None or not self.btnCommit.isChecked():
+        if self.graph is None:
```

Comment:
Doesn't this button check prevent the analysis to re-run when it is already running?